cli/resource/parse.go

```diff
@@ -78,6 +78,31 @@ func Parse(in io.Reader) ([]*types.Wrapper, error) {
 	return resources, err
 }
 
+// splitResources scans the content of the reader and splits the resources.
+// The resources should be separated by a line containing only "---".
+// An error will be returned if the
```

Comment:
Looks like you didn't quite finish the thought here :)

Reply:
Oops 😊